AntennaTracker/sensors.cpp

```diff
@@ -14,7 +14,6 @@ void Tracker::update_ahrs()
 void Tracker::update_compass(void)
 {
     if (AP::compass().enabled() && compass.read()) {
```

Comment:
So nothing within the brackets?

Reply:
Fixed.  We just call read() now and it does the enabled check.